pandas/tests/io/test_pickle.py

```diff
@@ -376,6 +376,40 @@ def test_read_infer(self, ext, get_random_path):
 
             tm.assert_frame_equal(df, df2)
 
+    @pytest.mark.parametrize("cruft", ["__MACOSX/", ".DS_STORE"])
+    def test_load_zip_with_hidden_folders(self, cruft, get_random_path):
+        """Test loading .zip files that have extraneous metadata in hidden folders. """
+        base = get_random_path
+        path1 = f"{base}.raw"
+        path2 = f"{base}.zip"
+        dummy = f"{base}.dummy"
+        compression = "zip"
+
+        with tm.ensure_clean(path1) as p1, tm.ensure_clean(
+            path2
+        ) as p2, tm.ensure_clean(dummy) as d:
+            df = tm.makeDataFrame()
+
+            # write to uncompressed file
```

Comment:
You can remove all of these comments

Reply:
Really? I'm just following the style of other tests in this file. I've made the comments more succinct but wouldn't want to remove them altogether as this is a bit of a weird edge-case to test.